test/system/045-start.bats

```diff
@@ -27,7 +27,10 @@ load helpers
     run_podman wait $cid_none_implicit $cid_none_explicit $cid_on_failure
 
     run_podman rm $cid_none_implicit $cid_none_explicit $cid_on_failure
-    run_podman stop -t 1 $cid_always
+    run_podman 0+w stop -t 1 $cid_always
+    if ! is_remote; then
```

Comment:
The warning is never seen in podman-remote. Should it be?

Reply:
Ideally yes but it's technically very difficult.  `logrus` logs are always shown on the server side, and never on the client side.

Reply:
Ack, thanks.